packages/react-client/src/hooks/screenShare.ts

```diff
@@ -14,7 +14,8 @@ export const useScreenShare = (): ScreenshareApi => {
   const ctx = useFishjamContext();
 
   const [state, setState] = ctx.screenshareState;
-  const tsClient = ctx.state.client.getTsClient();
+  const { fishjamClientRef } = useFishjamContext();
+  const tsClient = useMemo(() => fishjamClientRef.current, [fishjamClientRef]);
```

Comment:
Is this memo required here? 🤔

Reply:
reasumując wszystkie aspekty kwintesencji tematu, dochodzę do fundamentalnej konkluzji - nie jest